### ISSUE TITLE
Invalid data item 

### ISSUE THREAD

**Issue by Sksskksl** [ubershmekel/redditp#115]
Page said invalid data item (red box in upper right corner popped up twice with the text, clicking led here) and wouldn't load the next picture. Could move backwards to the ones I'd already seen but not forwards to new ones. Everything else seemed to work.

**Comment by Sksskksl** [comment]
On Android device, Chrome browser.

**Comment by ubershmekel** [comment]
Do you have a link to where this error happens? Does it still happen?

**Comment by Sksskksl** [comment]
No. It was on redditp r/tumblr. It doesn't happen often at all but it said to notify about it here. It worked alright the next day.

**Comment by ubershmekel** [comment]
Great. Thank you. If it does happen again - please let me know on this ticket.
